eng/pipelines/corefx-base.yml

```diff
@@ -198,19 +198,14 @@ jobs:
                 msbuildScript: $(_msbuildCommand)
                 framework: $(_framework)
                 outerloop: $(_outerloop)
-                enableAzurePipelinesReporter: false
 
                 ${{ if eq(parameters.isOfficialBuild, 'true') }}:
-                  isExternal: false
```

Comment:
Why are we removing this? isExternal is needed for public builds.

Reply:
Or is it defaulted to true when `helixToken != ''`?

Reply:
no it's not: https://github.com/dotnet/arcade/blob/851e36df83d3361e4bd8a70a2a8a89f762469f9a/eng/common/templates/steps/send-to-helix.yml#L24

Reply:
Ah right. Ok cool.